### DISCUSSION TITLE
RTL-SDR Blog V4 not working

### DISCUSSION THREAD

**Discussion by swainstm** [DSheirer/sdrtrunk#1869] [Q&A]
I believe release 0.6.0 was supposed to support RTL-SDR Blog V4 dongles. However, sdrtrunk does not seem to recognise my dongle. I am running Windows 11. If I am following youtube instructions correctly, waterfall should start immediately when you start the application?

Dongle works fine in SDR#.

Any ideas on how to troubleshoot and resolve this.

**Comment by funman1steve** [comment]
I run like 6 of them on one system.
It's definitely supported. Did you serialize all your dongles before attempting to use them, and did you use ZDiag to install the correct drivers first?

**Reply by swainstm** [reply]
I only have one. So do I need to serialise them? I did notice somewhere that its serial was 00000001.

Are you using Windows 11?

**Comment by funman1steve** [comment]
No you don't need to serialize if you only have one
but did you use ZDIAG to install the drivers as needed?

**Comment by swainstm** [comment]
Sorry I did not answer that question, but if you mean did I run ZADIG, yes I did. And with this driver and SDR#, the dongle works perfectly in that application. Or is there a ZDIAG that I am missing.

But that driver does not seem to like SDRTrunk for some reason. I found a log file with the following snipit 👎 

20240308 010255.040 [main] INFO  i.g.d.s.t.manager.TunerManager - Discovering tuners ...  [31MB/1GB 3%]
20240308 010255.123 [main] INFO  i.g.d.s.t.manager.TunerManager - LibUsb API Version: 1.0.262  [32MB/1GB 3%]
20240308 010255.124 [main] INFO  i.g.d.s.t.manager.TunerManager - LibUsb Version: 1.0.22.11312  [32MB/1GB 3%]
20240308 010255.140 [main] INFO  i.g.d.s.t.manager.TunerManager - LibUsb - discovered [9] potential usb devices  [32MB/1GB 3%]
20240308 010255.144 [main] INFO  i.g.d.s.t.manager.TunerManager - Discovered tuner at USB Bus [1] Port [2.6] Tuner Class [RTL-2832]  [34MB/1GB 3%]
20240308 010255.145 [main] INFO  i.g.d.s.t.manager.TunerManager - Tuner: USB Tuner - RTL-2832 USB Bus:1 Port:2.6 - Added / Starting ...  [34MB/1GB 3%]
20240308 010255.182 [main] ERROR i.g.d.s.t.u.USBTunerController - Can't open USB tuner - check driver or Linux udev rules  [9MB/56MB 16%]
20240308 010255.184 [main] INFO  i.g.d.s.t.m.DiscoveredTuner - Tuner Error - Stopping - RTL-2832 USB Bus:1 Port:2.6 Error: Can't open USB tuner - reinstall driver? - LIBUSB_ERROR_NOT_FOUND  [9MB/56MB 16%]
20240308 010255.184 [main] INFO  i.g.d.s.t.m.DiscoveredTuner - Stopping Tuner: RTL-2832 USB Bus:1 Port:2.6  [9MB/56MB 16%]
20240308 010255.184 [main] ERROR i.g.d.s.t.m.DiscoveredUSBTuner - Unable to start tuner [RTL-2832] - error: Can't open USB tuner - reinstall driver? - LIBUSB_ERROR_NOT_FOUND  [9MB/56MB 16%]
20240308 010255.186 [main] INFO  i.g.d.s.t.manager.TunerManager - LibUsb Hotplug event notification Is Not Supported on this platform.  [9MB/56MB 17%]

So sounds like there is some sort of issue with SDRTrunk not liking the driver on my Windows 11 PC?

The details of the driver that is loaded are : 
Driver Provider : libwdi
Driver date : 2/06/2012
Driver Version : 6.1.7600.16385
Digital Signer : USB\VID_0BDA&PID_2838&MI_00 (libwdi autogenerated)

The Driver Details : 
C:\Windows\system32\DRIVERS\winusb.sys
Provider : Microsoft Corporation
File Versions : 10.0.22621.2504 (WinBuild.160101.0800)

But driver seems to work on other programs, so I assume it is somehow related to something else.

**Comment by swainstm** [comment]
Kept playing around, and seem to have stumbled on the cause of the problem. I have a Dell XPS laptop plugged into a Thunderbolt dock. The RTL-SDRv4 was plugged into a USB port in the dock. However, if use a different USB hub in the device, suddenly it all starts working. So the issue seems to be a combination of SDRTrunk and the dell thunderbolt dock that is causing the issue. 

There is no change in drivers with this change, so the driver is the same.

**Comment by funman1steve** [comment]
How odd...
Glad you got it working :)
